codice/ddf
# DDF 2490 replace documentation with new outline

#### What does this PR do?

- Replaces documentation with new outline.
- Removes outdated contents files from old outline.
- Minor cleanup and formatting.

#### Who is reviewing it?

@mcalcote @vinamartin @Lambeaux @jrnorth 
#### Select at least one member from relevant component team(s) from below.

[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@coyotesqrl
@figliold
@kcwire
@lessarderic
@pklinef
@shaundmorris
@stustison

#### How should this be tested?

Build: 
- Builds successfully.
See attached pdf. 
- no missing sections: (search for `unresolved directive` in finished html or pdf.

#### Any background context you want to provide?

Sections 12-13 (Installing, Configuring, Securing) are part of another PR: https://github.com/codice/ddf/pull/1594, so those are lower priority for this PR.

#### What are the relevant tickets?
[DDF-2490](https://codice.atlassian.net/browse/DDF-2490)

#### Screenshots (if appropriate)

[documentation.pdf](https://github.com/codice/ddf/files/743588/documentation.pdf)

#### Checklist:
